eng/scripts/Language-Settings.ps1

```diff
@@ -4,31 +4,27 @@ $PackageRepository = "NPM"
 $packagePattern = "*.tgz"
 $MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/js-packages.csv"
 $BlobStorageUrl = "https://azuresdkdocs.blob.core.windows.net/%24web?restype=container&comp=list&prefix=javascript%2F&delimiter=%2F"
+$PackageRegex = "/\@(.*)\//i"
```

Comment:
Should these be global variables? If they are only used in the one function I would put them inline so they don't end up conflicting with other things in the global space.